annotation-client/src/test/resources/subtitle/minimal_qt_right.json

```diff
@@ -4,7 +4,8 @@
     "type": "FullTextResource",
     "language": "it",
     "value": "{QTtext}{timescale:1000}{font:Arial}{size:16}{backColor:0,0,0}{textColor:65535,65535,65535}{width:420}{justify:center}{dropShadow:on}{dropShadowOffset:1,1}{dropShadowTransparency:200}\n[00:00:01.250]\n[Dotsub Any Video Any Language]",
-    "format": "video/quicktime"
+    "format": "video/quicktime",
```

Comment:
Is this file needed? the minimal.json has exactly the purpose to have all mandatory fields. 
The caption/minimal.json needs to be updated as well. All integration tests in the annotation-client must pass after making changes into the code

Reply:
Improved.